### PR TITLE
Update cross-tenant-onedrive-migration.md

### LINKED ISSUE #11624: Wrong link
In this page I notice there is something wrong.
In the passage 
Prerequisites and settings
Microsoft SharePoint Online Powershell. Confirm you have the most recent version installed. [Download SharePoint Online Management Shell from Official Microsoft Download Center](https://learn.microsoft.com/en-us/download/details.aspx?id=35588)
I opened this link, but the last prompt was a 404 page. But when I search on the bing,I notice the link this [https://www.microsoft.com/en-us/download/details.aspx?id=35588](https://www.microsoft.com/en-us/download/details.aspx?id=35588)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3c59d15b-a5d1-400a-aa12-8d768817f8f6
* Version Independent ID: 3c59d15b-a5d1-400a-aa12-8d768817f8f6
* Content: [Cross-tenant OneDrive migration overview - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-onedrive-migration?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/cross-tenant-onedrive-migration.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/cross-tenant-onedrive-migration.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the download URL

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11624

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/cross-tenant-onedrive-migration.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Microsoft 365 Business Basic/Business Standard/Business Premium/F1/F3/E3/A3/E5/A
 
 ## Prerequisites and settings
 
-- **Microsoft SharePoint Online Powershell**. Confirm you have the most recent version installed. [Download SharePoint Online Management Shell from Official Microsoft Download Center](/download/details.aspx?id=35588)
+- **Microsoft SharePoint Online Powershell**. Confirm you have the most recent version installed. [Download SharePoint Online Management Shell from Official Microsoft Download Center](https://www.microsoft.com/en-us/download/details.aspx?id=35588)
 
 - **Turn off service encryption with Customer Key enabled.** Confirm that the source OneDrive tenant **doesn't** have Service encryption with Microsoft Purview Customer Key enabled. If enabled on Source tenant, the migration will fail. [Learn more on Service encryption with Microsoft Purview Customer Key](/microsoft-365/compliance/customer-key-overview)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-onedrive-migration.md]:
```suggestion
- **Microsoft SharePoint Online Powershell**. Confirm you have the most recent version installed. [Download SharePoint Online Management Shell from the official Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=35588).

- **Turn off service encryption with Customer Key enabled.** Confirm that the source OneDrive tenant **doesn't** have Service encryption with Microsoft Purview Customer Key enabled. If enabled on the source tenant, the migration will fail. [Learn more on Service encryption with Microsoft Purview Customer Key](/microsoft-365/compliance/customer-key-overview).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoanneHendrickson Please merge this PR. Thanks

**Pull request merged**